src/test/scala/nl.knaw.dans.easy.bagstore/server/BagsServletSpec.scala

```diff
@@ -67,7 +67,7 @@ class BagsServletSpec extends TestSupportFixture
   "get" should "enumerate the bags in all bag-stores" in {
     get("/") {
       status shouldBe 200
-      body.lines.toList should contain only(
+      body.linesIterator.toList should contain only(
```

Comment:
Why do you switch from `body.lines` to `body.linesIterator`? I thought the latter was deprecated since Scala 2.11?

Reply:
It was deprecated in Scala 2.11 but brought back in 2.12 due to conflicts with java's own `String#lines` method. https://github.com/scala/bug/issues/11125#issuecomment-423231998